gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -67,7 +63,7 @@ The following minimum requirements apply to a small Kubernetes cluster used for
 
 * Support for the creation of a `Service` resource of the `LoadBalancer` type.
 +
-To avoid deployment errors when using this guide, your cluster must support this process. However, Flex Gateway supports these Ingress Controller services: `ClusterIP`, `NodePort`, `LoadBalancer` (default), `ExternalName`. 
+To avoid deployment errors when using this guide, your cluster must support this default process. However, note that Flex Gateway supports these services: `ClusterIP`, `NodePort`, `LoadBalancer` (default), `ExternalName`. 
```

Comment:
I'm not sure if we can call this a "process". Also, the LoadBalancer type is the default option because we defined it so when creating the helm chart, but it's not a kubernetes default. I suggest something like this:

```suggestion
To avoid deployment errors when using this guide, your cluster must support creating a services of type `LoadBalancer`. However, note that the Service type can be customized when deploying (maybe we can add a link to the deployment section). Flex Gateway supports these service types: `ClusterIP`, `NodePort`, `LoadBalancer` (default), `ExternalName`. 
```